conversion-measurement/functions/apps/adtech.js

```diff
@@ -14,106 +14,109 @@
  * limitations under the License.
  */
 
+const functions = require('firebase-functions');
 const express = require('express')
 const cookieParser = require('cookie-parser')
 require('dotenv').config({ path: `.env.${process.env.NODE_ENV}` })
+const http = require('http')
 
-const app = express()
-app.use(express.json())
-app.use(cookieParser())
+const adtech = express()
+adtech.use(express.json())
+adtech.use(cookieParser())
 
-app.set('view engine', 'pug')
-const PORT = 8084
-const demoHomeUrl = process.env.DEMO_HOME_URL
-const publisherUrl = process.env.PUBLISHER_URL
-const advertiserUrl = process.env.ADVERTISER_URL
+adtech.set('view engine', 'pug')
+adtech.set('views', './views/adtech')
 const adtechUrl = process.env.ADTECH_URL
 
-app.use(express.static('static'))
+// app.use(express.static('static'))
 
-app.get('/', (req, res) => {
+adtech.get('/', (req, res) => {
   res.render('index')
 })
 
 /* -------------------------------------------------------------------------- */
 /*                               Debugging setup                              */
 /* -------------------------------------------------------------------------- */
 
-app.use(function(req, res, next) {
-  // Optional: set a regular measurement 3P cookie
-  const legacyMeasurementCookie = req.cookies['measure']
+adtech.use(function(req, res, next) {
+  console.log('Time:', Date.now(), " ", req.originalUrl, " Cookies: ", req.cookies)
+    // Optional: set a regular measurement 3P cookie
+  var headers = []
+  const legacyMeasurementCookie = req.cookies['__session']
   if (legacyMeasurementCookie === undefined) {
     const cookieValue = Math.floor(Math.random() * 1000000000000000)
-    res.set(
-      'Set-Cookie',
-      `measure=${cookieValue}; SameSite=None; Secure; HttpOnly`
-    )
+    headers.push(`__session=${cookieValue}; SameSite=None; Secure; HttpOnly`)
   }
 
   // Set the Attribution Reporting debug cookie
   const debugCookie = req.cookies['ar_debug']
   if (debugCookie === undefined) {
-    res.set('Set-Cookie', 'ar_debug=1; SameSite=None; Secure; HttpOnly')
+    headers.push('ar_debug=1; SameSite=None; Secure; HttpOnly')
```

Comment:
Question: 
IIRC there were some issues on debug cookie setting in Firebase, is this resolved? As in, will `ar_debug` be set correctly?
If yes, IIUC this means the cookie name can be anything set, so what's the reason to rename  the measurement cookie above to `_session`?

Reply:
yes outgoing headers will be set correctly, the issues is about cookies received on the server end where I now use `__session` instead of `measurement` cookie.

Reply:
Understood ty, nit: Do you think we could add a comment above the `__session=` line to mention this, as to not confuse folks at to why this cookie is called `__session`?